donor-portal/src/components/navbar/index.tsx

```diff
@@ -22,32 +22,33 @@ export default function NavBar() {
         <div className="logo" />
       </Link>
       <div className="nav-links">
-        <Link to="/about-us" className="nav-link">
+        <NavLink to="/" className="nav-link" exact>
+          Home
+        </NavLink>
```

Comment:
Nice! This should make styling easier. 👍🏽 
And removing spans cleans things up.

Reply:
Yh those | spans were cluttering things up